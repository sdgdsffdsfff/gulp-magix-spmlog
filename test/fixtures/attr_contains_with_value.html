<a href="http://api.test.com"></a>
<a href="http://open.test.com"></a>