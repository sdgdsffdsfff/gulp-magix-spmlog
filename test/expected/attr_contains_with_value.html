<a href="http://api.test.com" data-spm-click="gostr=/*.*.*;locaid=*"></a>
<a href="http://open.test.com"></a>